core/integer/pow_spec.rb

```diff
@@ -19,13 +19,13 @@
       2.pow(61, 5843009213693951).should eql 3697379018277258
       2.pow(62, 5843009213693952).should eql 1551748822859776
       2.pow(63, 5843009213693953).should eql 3103497645717974
-      2.pow(64, 5843009213693954).should eql  363986077738838
+      2.pow(64, 5843009213693954).should eql 363986077738838
     end
 
     it "handles sign like #divmod does" do
-       2.pow(5,  12).should ==  8
-       2.pow(5, -12).should == -4
-      -2.pow(5,  12).should ==  4
+      2.pow(5, 12).should == 8
+      2.pow(5, -12).should == -4
+      -2.pow(5, 12).should == 4
       -2.pow(5, -12).should == -8
```

Comment:
Rubocop trips here because the ` 2` and `-2` are aligned. I just brutally ignored that, do we want a special case for this? I think there is a Rubocop setting for this, but I have to look up which one it is.

Reply:
It's good as-is, I like simple alignment, much easier to maintain